docs/docs/modules/indexes/vector_stores/integrations/typeorm.mdx

```diff
@@ -0,0 +1,81 @@
+---
+sidebar_class_name: node-only
+---
+
+# TypeORM
+
+For augmenting existing models in a PostgreSQL database with vector search, Langchain supports using [TypeORM](https://typeorm.io/) together with PostgreSQL and the [`pgvector`](https://github.com/pgvector/pgvector) Postgres extension.
```

Comment:
We're not really augmenting an existing model though right? This would be creating a new model in a database?

Reply:
indeed, I change this